bi/pkg/cluster/kind/net.go

```diff
@@ -7,7 +7,7 @@ import (
 	"log/slog"
 	"net"
 
-	"github.com/dghubble/ipnets"
+	"github.com/apparentlymart/go-cidr/cidr"
```

Comment:
We were already using this `cidr` package elsewhere so remove the `ipnets` module that's only used here as the functionality of `cidr` is a superset.